cluster/examples/kubernetes/ceph/rook-operator-config.yaml

```diff
@@ -0,0 +1,22 @@
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: rook-operator-config
+  namespace: rook-ceph
```

Comment:
This needs to be the same as operator namespace? if yes please add a comment here

Reply:
Yes. Added a comment.